src/components/page/page.comp.tsx

```diff
@@ -52,6 +54,12 @@ const PageComp = ({ context }: IProps) => {
   const [queryParams, setQueryParams] = useState<IConfigInputField[]>(getAllConfig?.queryParams || []);
   const [items, setItems] = useState<any[]>([]);
   const [filter, setFilter] = useState<string>('');
+  const [dataPage, setDataPage] = useState<number>(parseInt(paginationConfig?.params.page.value || '1'));
```

Comment:
Why no having one state for all pagination params rather than one per property?

Reply:
Because I figured they might have effect on different components in the future ? Like where the total items number would be displayed in lazy-loading, like the buttons display, on the query params component if we display some of them, etc.

Reply:
IMHO, all pagination params should be in a single state object.

Reply:
They are refactored into a single pagination state now.